mas_perception_libs/ros/include/mas_perception_libs/image_bounding_box.h

```diff
@@ -39,8 +40,13 @@ class ImageBoundingBox
     std::vector<std::vector<cv::Point2f>> mBoxVerticesVector;
 };
 
-cv::Mat
-cropImage(cv::Mat &image, std::vector<cv::Point2f> &vertices);
+/*!
+ * @brief Draw boxes on a sensor_msgs/Image message using BoundingBox2D objects. Call drawLabeledBoxes() defined in
+ *        bounding_box_2d.h in the background.
+ */
+sensor_msgs::ImagePtr
+drawLabeledBoxesImgMsg(const sensor_msgs::Image&, std::vector<BoundingBox2D>,
```

Comment:
here are you modifying the sensor_msgs::Image? in that case const would be a problem. And the vector should probably be const ref.